src/main/java/dev/emi/trinkets/data/EntitySlotLoader.java

```diff
@@ -72,25 +64,22 @@ protected Map<String, Map<String, Set<String>>> prepare(ResourceManager resource
 								JsonArray assignedSlots = JsonHelper.getArray(jsonObject, "slots", new JsonArray());
 								Map<String, Set<String>> groups = new HashMap<>();
 
-								if (assignedSlots != null) {
```

Comment:
What is the purpose of removing these null checks?

Reply:
The reason why I removed those null checks is that `JsonHelper.getArray` is never null, it either has an assigned value or uses the default fallback value.